src/operations/blas2/trsv.hpp

```diff
@@ -0,0 +1,142 @@
+/***************************************************************************
+ *
+ *  @license
+ *  Copyright (C) Codeplay Software Limited
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  For your convenience, a copy of the License has been included in this
+ *  repository.
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *  SYCL-BLAS: BLAS implementation using SYCL
+ *
+ *  @filename trsv.hpp
+ *
+ **************************************************************************/
+
+#ifndef TBSV_HPP
+#define TBSV_HPP
+#include "operations/blas2_trees.h"
+#include "operations/blas_operators.hpp"
```

Comment:
Do you need this header here or any of the headers below this one.

Reply:
I removed them!